quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-with-nat-gateway/main.bicep

```diff
@@ -159,7 +161,4 @@ resource workspace 'Microsoft.Databricks/workspaces@2024-05-01' = {
       }
     }
   }
-  dependsOn: [
-    nsg
```

Comment:
why are we removing this? This is going to impact reality of arm template deployment.